test/build.bats

```diff
@@ -43,6 +43,11 @@ load common
         echo "$output"
         [[ $status -eq 0 ]]
         [[ $output = *'sage:'* ]]
+        # Executables print usage when run without arugments
+        run "$i"
+        echo "$output"
+        [[ $status -eq 0 ]]
+        [[ $output = *'sage:'* ]]
```

Comment:
For the first question, this behavior is because of some arcane GNU argparse administrivia. 
To print the full help text in the case of no arguments, one does:
https://github.com/hpc/charliecloud/blob/ee62ef321ece920e359c8f72b54681ee98e32e38/bin/ch-run.c#L442-L443

`ARGP_HELP_STD_HELP` prints the full help then exits with success. `ARGP_HELP_STD_USAGE` prints a message and exits with failure, but it only shows a partial message. All the other options outlined [here](https://www.gnu.org/software/libc/manual/html_node/Argp-Help-Flags.html) print a partial message.

Reply:
For the second, I based the test off of the --help test. The *sage catches `Usage` and `usage`. In this case, it's important since argparse.print_help which is used in all of the Python programs prints `usage` while everything else prints `Usage`. This could be changed.

Reply:
Makes sense. Resolving.